src/Countries/US/NewJersey/NewJerseyFamilyMedicalLeave/V20200101/NewJerseyFamilyMedicalLeave.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\NewJersey\NewJerseyFamilyMedicalLeave\V20200101;
+
+use Appleton\Taxes\Countries\US\NewJersey\NewJerseyFamilyMedicalLeave\NewJerseyFamilyMedicalLeave as BaseNewJerseyFamilyMedicalLeave;
+use Appleton\Taxes\Traits\HasWageBase;
+use Illuminate\Database\Eloquent\Collection;
+
+class NewJerseyFamilyMedicalLeave extends BaseNewJerseyFamilyMedicalLeave
+{
+    use HasWageBase;
+
+    const TAX_RATE = 0.0016;
+    const WAGE_BASE = 134900;
```

Comment:
wage base was 34400 in 2019 and the AC doesn't say anything about it changing, or is that the one you think may have been wrong initially?

Reply:
The AC does say to change it. I assumed thats what it meant by "family-leave insurance programs. Or am I wrong? AC 1. > The wage base for employee contributions to the temporary disability insurance and family-leave insurance programs is to be $134,900 in 2020

Reply:
gotcha, I think you're right

Reply:
I wasn't 100% which is why I posted the AC. I wanted the reviewers to be as informed as possible in case I misread it